mlos_bench/mlos_bench/config/schemas/tunables/tunable-params-schema.json

```diff
@@ -120,6 +126,12 @@
                     "minItems": 2,
                     "maxItems": 2
                 },
+                "quantization": {
+                    "type": "number"
```

Comment:
I guess it's only an int for "integer" types:
https://automl.github.io/ConfigSpace/main/api/hyperparameters.html#module-ConfigSpace.api.types.integer

Reply:
it is `int` for `ConfigSpace`, but JSON schema does not support ints and floats - the only valid type is `number` 😒

Reply:
json has "integer" too:
https://json-schema.org/understanding-json-schema/reference/numeric#integer